src/SourceBuild/content/eng/Versions.props

```diff
@@ -12,20 +12,14 @@
 
   <PropertyGroup>
     <!--
-      Building .NET from source depends on one or two tar.gz files depending on the branch's current
+      Building .NET from source depends on several archives, depending on the branch's current
       source-buildability status.
 
-      PrivateSourceBuiltArtifactsPackageVersion is a tar.gz of .NET build outputs from a previous
-      build needed to build the current version of .NET. This is always defined, because .NET needs
-      to be bootstrappable at any point in time.
-
-      PrivateSourceBuiltPrebuiltsPackageVersion is a tar.gz of assets downloaded from the internet
-      that are needed to build the current version of .NET. Early in the lifecycle of a .NET major
-      or minor release, prebuilts may be needed. When the release is mature, prebuilts are not
-      necessary, and this property is removed from the file.
+      These URLs can't be composed from their base URL and version as we read them from the
+      prep.sh and pipeline scripts, outside of MSBuild.
     -->
-    <PrivateSourceBuiltArtifactsPackageVersion>0.1.0-8.0.100-5.centos.8-x64</PrivateSourceBuiltArtifactsPackageVersion>
-    <PrivateSourceBuiltPrebuiltsPackageVersion>0.1.0-8.0.100-8.centos.8-x64</PrivateSourceBuiltPrebuiltsPackageVersion>
-    <PrivateSourceBuiltSDKVersion>8.0.100-alpha.1.23080.1-centos.8-x64</PrivateSourceBuiltSDKVersion>
+    <PrivateSourceBuiltArtifactsUrl>https://dotnetcli.azureedge.net/dotnet/source-built-artifacts/assets/Private.SourceBuilt.Artifacts.0.1.0-8.0.100-5.centos.8-x64.tar.gz</PrivateSourceBuiltArtifactsBaseUrl>
+    <PrivateSourceBuiltPrebuiltsUrl>https://dotnetcli.azureedge.net/dotnet/source-built-artifacts/assets/Private.SourceBuilt.Prebuilts.0.1.0-8.0.100-8.centos.8-x64.tar.gz</PrivateSourceBuiltPrebuiltsUrl>
+    <PrivateSourceBuiltSdkUrl>https://dotnetcli.azureedge.net/dotnet/source-built-artifacts/sdks/dotnet-sdk-8.0.100-preview.1.23115.1-centos.8-x64.tar.gz</PrivateSourceBuiltSdkUrl>
```

Comment:
This one feels like it should have the rid in it's name.  I question if the Artifacts should also have the rid in it's name.  These two are always rid specific.  The Prebuilts are more "portable"

Reply:
In order to keep prep.sh simpler, I left the RID off of the artifacts for the moment.